build/yaml/crd/vpc/crd.nsx.vmware.com_ipaddressallocations.yaml

```diff
@@ -63,8 +64,8 @@ spec:
           status:
             description: IPAddressAllocationStatus defines the observed state of IPAddressAllocation.
             properties:
-              CIDR:
-                description: CIDR is the allocated CIDR
+              allocation_ips:
+                description: AllocationIPs is the allocated AllocationIPs
```

Comment:
`allocation_ips is the allocated IP addresses`

Reply:
ok